README.md

```diff
@@ -1,7 +1,6 @@
 # laminas-cache-storage-adapter-{adapter}
 
-[![Build Status](https://travis-ci.com/laminas/laminas-cache-storage-adapter-{adapter}.svg?branch=master)](https://travis-ci.com/laminas/laminas-cache-storage-adapter-{adapter})
-[![Coverage Status](https://coveralls.io/repos/github/laminas/laminas-cache-storage-adapter-{adapter}/badge.svg?branch=master)](https://coveralls.io/github/laminas/laminas-cache-storage-adapter-{adapter}?branch=master)
+[![Build Status](https://github.com/laminas/laminas-cache-storage-adapter-{adapter}/workflows/continuous-integration.yml/badge.svg)](https://github.com/laminas/laminas-cache-storage-adapter-{adapter}/actions/workflows/continuous-integration.yml)
```

Comment:
There is `/actions/` missing for the SVG. I've had the same problem recently in another package.